### ISSUE TITLE
Bug: `KeyboardInterrupt` raised when pasting FINN code from console

### ISSUE THREAD

**Issue by seemir** [seemir/stressa#141]
**Summary:**
When pasting Finn codes from previous searches printed to the console a `KeyboardInterrupt` gets raised.  

**Steps to Reproduce:**
1. Start application `python main.py` 
2. Search Finn code from finn.no
3. Check log from console and make sure that the full log gets printed to console
4. Restart application
5. Copy the Finn code from the log output in the console and search

**Expected Results:**
Finn ad information should be displayed in application

**Actual Results:**
Application crashes and the following message gets printed to console:
```
Traceback (most recent call last):
  File "C:\Users\samir\Dropbox\BACKUP\github\stressa\source\ui\models\finn_model.py", line 75, in <lambda>
    lambda: self.clear_finn_info("_1"))
KeyboardInterrupt
```

**Comment by seemir** [comment]
The bug was thrown because one copies strings directly from `console` using `Ctrl` + `C` and `Ctrl + V`. This is not necessary as by default the option `Copy on select` is chosen on GitBash. Thus its enough to just select the text one wants to copy. Still, have added an `except_hook` to catch all exceptions that not captured by any logic in application. The exceptions are thus displayed to the end-user using `ErrorView`, see Issue  #142 for further information on how this was solved.

**Comment by seemir** [comment]
Closed by 9e9aafa8914dc5daedf7ce281cc7948630e3f98f
